app/js/components/mixins/ModalEffects.react.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+let ModalEffects = InnerComponent => class extends React.Component {
```

Comment:
Higher order functions are for .map, filter, reduce and what have you. That contains another function, but having a function that contains a class is a no no. Though inbetween javascript converts es6 classes to function. It should never be used inside classes.